sigmastate/src/main/scala/sigmastate/lang/syntax/Literals.scala

```diff
@@ -15,81 +14,87 @@ trait Literals { l =>
   import builder._
   def atSrcPos[A](parserIndex: Int)(thunk: => A): A
   def srcCtx(parserIndex: Int): SourceContext
-  def Block: P[Value[SType]]
-  def Pattern: P0
+  
+  def Block[_:P]: P[Value[SType]]
```

Comment:
Scaladocs missed in the trait

Reply:
done in https://github.com/ScorexFoundation/sigmastate-interpreter/commit/6d2feabfc29a54eb108b77dd09ecf51f51fdf3f1